README.md

```diff
@@ -230,13 +230,44 @@ the multiswitch target in the manifest. For example, have a look at the
 [manifest](examples/multiswitch.p4app/p4app.json) for the multiswitch example
 app.
 
+
+#### Specifying entries for each switch
+The `ipv4_lpm`, `send_frame` and `forward` tables are automatically populated
+with this target. Additionally, you can specify entries to be added to each
+switch. You can either include a commands file, or an array of entries. For
```

Comment:
What happens if entries overlap between the file and the specific entries? Which one has precedence?